setup/web.sh

```diff
@@ -36,6 +36,12 @@ tools/editconf.py /etc/nginx/nginx.conf -s \
 tools/editconf.py /etc/php5/fpm/pool.d/www.conf -c ';' \
 	pm.max_children=8
 
+# Enable PHP's slowlog to better diagnose slow running scripts
+tools/editconf.py /etc/php5/fpm/pool.d/www.conf -c ';' \
+	slowlog=/var/log/php5-fpm.log.slow
```

Comment:
Just curious, why not name it `php5-fpm.slow.log`?  It is, after all, a .log file.

Just wondering if this convention is used elsewhere.  Apologies if this is bikeshedding.  :)